### DISCUSSION TITLE
Additional Encoder Options (Opus/Vorbis)

### DISCUSSION THREAD

**Discussion by theksepyro** [advplyr/audiobookshelf#3282] [Ideas]
The ability to merge a book into m4b files is a great feature to have. I think it would be nice to also have the option to merge/encode into ogg (preferably opus, but possibly vorbis) files.. This conforms with "open" spirit of  the project

**Comment by nichwall** [comment]
You can specify the codec in the encoding options.

![Screenshot_20240814-080932.png](https://github.com/user-attachments/assets/8c37579c-9eb6-4c88-a6c4-a9ee8c1ef87c)

**Reply by theksepyro** [reply]
I see!

 I guess what confuses me about that is that if the output was ogg opus, it would no longer be an "m4b encode" like the button says. I'll give it a try when I get home this evening.

Thanks!

**Reply by advplyr** [reply]
M4B can have Opus codec. 
I would like to improve the encoder as well. There are a few feature requests open for that.

**Reply by theksepyro** [reply]
Ahhh, that's the missing piece that clears it up for me. Thanks!
